src/nvim/testdir/test_breakindent.vim

```diff
@@ -12,53 +12,85 @@ source view_util.vim
 
 let s:input ="\tabcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOP"
 
-function s:screen_lines(lnum, width) abort
+func s:screen_lines(lnum, width) abort
   return ScreenLines([a:lnum, a:lnum + 2], a:width)
-endfunction
+endfunc
 
-function! s:compare_lines(expect, actual)
+func! s:compare_lines(expect, actual)
   call assert_equal(join(a:expect, "\n"), join(a:actual, "\n"))
-endfunction
+endfunc
 
-function s:test_windows(...)
+func s:test_windows(...)
   call NewWindow(10, 20)
   setl ts=4 sw=4 sts=4 breakindent
   put =s:input
   exe get(a:000, 0, '')
-endfunction
+endfunc
 
-function s:close_windows(...)
+func s:close_windows(...)
   call CloseWindow()
   exe get(a:000, 0, '')
-endfunction
+endfunc
 
-function Test_breakindent01()
+func Test_breakindent01()
   " simple breakindent test
   call s:test_windows('setl briopt=min:0')
-  let lines=s:screen_lines(line('.'),8)
-  let expect=[
-\ "    abcd",
-\ "    qrst",
-\ "    GHIJ",
-\ ]
+  let lines = s:screen_lines(line('.'),8)
+  let expect = [
+	\ "    abcd",
+	\ "    qrst",
+	\ "    GHIJ",
+	\ ]
   call s:compare_lines(expect, lines)
   call s:close_windows()
-endfunction
+endfunc
+
+func Test_breakindent01_vartabs()
+  " like 01 but with vartabs feature
+  if !has("vartabs")
+    return
+  endif
+  call s:test_windows('setl briopt=min:0 vts=4')
+  let lines = s:screen_lines(line('.'),8)
+  let expect = [
+	\ "    abcd",
+	\ "    qrst",
+	\ "    GHIJ",
+	\ ]
+  call s:compare_lines(expect, lines)
+  call s:close_windows('set vts&')
+endfunc
 
-function Test_breakindent02()
+func Test_breakindent02()
   " simple breakindent test with showbreak set
   call s:test_windows('setl briopt=min:0 sbr=>>')
-  let lines=s:screen_lines(line('.'),8)
-  let expect=[
-\ "    abcd",
-\ "    >>qr",
-\ "    >>EF",
-\ ]
+  let lines = s:screen_lines(line('.'),8)
+  let expect = [
+	\ "    abcd",
+	\ "    >>qr",
+	\ "    >>EF",
+	\ ]
   call s:compare_lines(expect, lines)
   call s:close_windows('set sbr=')
-endfunction
+endfunc
+
+func Test_breakindent02_vartabs()
+  if !has("vartabs")
+    return
+  endif
+  " simple breakindent test with showbreak set
+  call s:test_windows('setl briopt=min:0 sbr=>> vts=4')
+  let lines = s:screen_lines(line('.'),8)
+  let expect = [
+	\ "    abcd",
+	\ "    >>qr",
+	\ "    >>EF",
+	\ ]
+  call s:compare_lines(expect, lines)
+  call s:close_windows('set sbr= vts&')
+endfunc
 
-function Test_breakindent03()
+func Test_breakindent03()
   " simple breakindent test with showbreak set and briopt including sbr
   call s:test_windows('setl briopt=sbr,min:0 sbr=++')
   let lines=s:screen_lines(line('.'),8)
```

Comment:
There was space added before,after `=`. Check nearby lines for whitespace changes.